hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -397,6 +397,10 @@ class TrainerParams(param.Parameterized):
         param.Number(default=None,
                      doc="PyTorch Lightning trainer flag 'limit_test_batches': Limit the test dataset to the "
                          "given number of batches if integer, or proportion of test dataset if float.")
+    pl_limit_batches: Optional[IntOrBool] = \
+        param.Number(default=False,
+                     doc="PyTorch Lightning trainer flag 'fast_dev_run': Runs n if set to ``n`` (int) else 1 if set to"
+                         "``True`` batch(es) of train, val and test to" "find any bugs (ie: a sort of unit test).")
```

Comment:
I see that this matches the description in PL's Trainer class, but I wonder if we can make it clearer by explaining that the default is False - i.e. use all batches? Also I find their statement 'to find any bugs (ie: a sort of unit test).' unnecessary - what do you think?

Reply:
Sounds great! I'll update that.